src/coreclr/jit/lowerxarch.cpp

```diff
@@ -6521,18 +6521,8 @@ bool Lowering::IsContainableImmed(GenTree* parentNode, GenTree* childNode) const
 // Note: if the tree oper is neither commutative nor a compare oper
 // then only op2 can be reg optional on xarch and hence no need to
 // call this routine.
-GenTree* Lowering::PreferredRegOptionalOperand(GenTree* tree)
+GenTree* Lowering::PreferredRegOptionalOperand(GenTree* op1, GenTree* op2)
```

Comment:
Can you update the arguments in the method summary?

Reply:
Must've missed this one, fixed.